mlflow/keras/__init__.py

```diff
@@ -52,8 +54,8 @@ def get_default_conda_env(include_cloudpickle=False):
         additional_conda_channels=None)
 
 
-def save_model(keras_model, path, conda_env=None, mlflow_model=Model(), custom_objects=None,
-               **kwargs):
+def save_model(keras_model, path, conda_env=None, keras_module=None, mlflow_model=Model(),
```

Comment:
Should we reorder `keras_module` to come after `mlflow_model` to avoid behavior changes in the case where people are passing these args as positional args?

Reply:
good point. I will update.